backend/src/main/scala/cromwell/backend/BackendJobExecutionActor.scala

```diff
@@ -54,5 +53,5 @@ trait BackendJobExecutionActor extends BackendJobLifecycleActor with ActorLoggin
   /**
     * Abort a running job.
     */
-  def abortJob: Future[JobAbortResponse]
+  def abortJob: Unit
```

Comment:
@mcovarr @Horneth Haha, ok, somebody mentioned it! 

If you guys would like, let's tech talk this. The questions would be:
a) what would be returned
b) what would you actually do with the result

In the Ye Olde world, we never check for this or do anything special as a result so it would have to be a new feature.